wasm/demo/package.json

```diff
@@ -24,7 +25,7 @@
         "build": "webpack",
         "dist": "webpack --mode production",
         "test": "cd ../tests; pipenv run pytest",
-        "ci": "start-server-and-test dev http-get://localhost:8080 test"
+        "ci": "start-server-and-test 'npm run build && serve dist/ -np 8080' :8080 test"
```

Comment:
It seems like it does not recognize the `-np` param in travis. Maybe we need a newer version there?

Reply:
No, I just thought it supported flag concatenation without checking, it should just be `-n -p 8080`.